src/components/playback/playbackmanager.js

```diff
@@ -1306,6 +1306,7 @@ class PlaybackManager {
                 return false;
             }
 
+            const container = mediaSource.Container;
```

Comment:
Can `mediaSource.Container` be combined like `mkv,webm`?
_I currently have just `mkv`._

Reply:
Inspired by https://github.com/jellyfin/jellyfin/blob/c795f17fa6ae9e4f68f23cb569a1129de8fd7635/MediaBrowser.Model/Dlna/ContainerProfile.cs#L19-L72